pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -892,6 +897,132 @@ var _ = Describe("Snapshotter", func() {
 				})
 			})
 		})
+
+		Describe("Scenarios to update full snapshot lease", func() {
+			var (
+				ssr                     *Snapshotter
+				lease                   *v1.Lease
+				FullSnapshotLeaseStopCh chan struct{}
+				ctx                     context.Context
+				cancel                  context.CancelFunc
+			)
+			BeforeEach(func() {
+				snapstoreConfig = &brtypes.SnapstoreConfig{Container: path.Join(outputDir, "default.bkp")}
+				store, err = snapstore.GetSnapstore(snapstoreConfig)
+				Expect(err).ShouldNot(HaveOccurred())
+
+				Expect(os.Setenv("POD_NAME", "test_pod")).To(Succeed())
+				Expect(os.Setenv("POD_NAMESPACE", "test_namespace")).To(Succeed())
+				lease = &v1.Lease{
+					TypeMeta: metav1.TypeMeta{
+						Kind:       "Lease",
+						APIVersion: "v1",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      brtypes.DefaultFullSnapshotLeaseName,
+						Namespace: os.Getenv("POD_NAMESPACE"),
+					},
+				}
+				snapshotterConfig := &brtypes.SnapshotterConfig{
+					FullSnapshotSchedule: fmt.Sprintf("%d %d * * *", 0, 0),
+				}
+				os.Setenv("UNIT_TEST", "true")
+				ssr, err = NewSnapshotter(logger, snapshotterConfig, store, etcdConnectionConfig, compressionConfig, healthConfig, snapstoreConfig)
+				Expect(err).ShouldNot(HaveOccurred())
+				ssr.PrevFullSnapshot = nil
+				ssr.K8sClientset = fake.NewClientBuilder().Build()
+				ssr.HealthConfig.SnapshotLeaseRenewalEnabled = true
+				FullSnapshotLeaseStopCh = make(chan struct{})
+			})
+			AfterEach(func() {
+				Expect(os.Unsetenv("POD_NAME")).To(Succeed())
+				Expect(os.Unsetenv("POD_NAMESPACE")).To(Succeed())
+				Expect(os.Unsetenv("UNIT_TEST")).To(Succeed())
```

Comment:
now, I guess you can remove this

Reply:
@ishan16696 Sorry, I forgot to remove those. Updated